src/bandersnatch/mirror.py

```diff
@@ -640,7 +641,10 @@ async def sync_release_files(self, package: Package) -> None:
         for release_file in package.release_files:
             try:
                 downloaded_file = await self.download_file(
-                    release_file["url"], release_file["digests"]["sha256"]
+                    release_file["url"],
+                    release_file["size"],
+                    isoparse(release_file["upload_time_iso_8601"]),
```

Comment:
I'm a big fan of keeping dependencies down. I feel native datetime can parse this from strftime - Can we do that to drop the dependency please?

Reply:
This is done in https://github.com/pypa/bandersnatch/pull/822/commits/880650fd9f156ebe83a8694031e92da86410b2a6#diff-8b844ef0ea8cbe96fbc0bd8fdfebc3b9e245509c5909654721be845486d0f82fR647